packages/react-storage/src/components/StorageBrowser/Views/utils.ts

```diff
@@ -16,7 +16,9 @@ export const listViewHelpers = ({
   disableRefresh: boolean;
   range: [start: number, end: number];
 } => {
-  const isLastPage = Math.round(resultCount / pageSize) === currentPage;
+  const isLastPage =
+    Math.round(resultCount / pageSize) === currentPage ||
+    Math.round(resultCount / pageSize) === 0;
```

Comment:
Think this makes sense but having a hard time reading the additional condition here as relating to the current page, since `0` isn't an allowed value of `currentPage`. Can we add a small comment explaining the logic?

Reply:
Yes, let me take another look. I just realized there are some other scenarios to test as well.

Reply:
I think I was overthinking it. I updated the existing logic to use `Math.ceil` instead of .round. I think then it should cover all the scenarios.